openstudiocore/src/utilities/core/Finder.hpp

```diff
@@ -103,7 +103,7 @@ namespace openstudio{
     typename std::vector<T>::const_iterator it;
     it = find_if(vec.begin(),vec.end(),finder);
     if (it != vec.end()) {
-      index = it - vec.begin();
+      index = (int)(it - vec.begin());

```

Comment:
Yes, `static_cast<int>(it - vec.begin())` is the "best" way to do it. You still are forcing one data type to be something different, but this is easy to search through the code for and find all the cases you are doing it.